bin/aws_pp.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status
+set -e
+
+# Installing necessary tool for the script: awscli
+pip install -qq awscli --upgrade
+
+# Current list of other regions we work with
+aws_regions=("eu-west-1" "eu-west-2" "eu-west-3" "eu-central-1" "ca-central-1" "us-east-1" "us-east-2" "us-west-1" "us-west-2")
+
+# We make sure to correctly set the source region with the default one (which should be eu-west-1 for KubeNow AWS builder),
+# the latest create AMI ID and its name
+aws_source_region="$AWS_DEFAULT_REGION"
+new_created_ami=$(grep -A 2 "successful builds" </tmp/pckr_build_log.txt | grep "^eu-west-1" | awk '{ print $2 }')
+ami_name=$(aws ec2 describe-images --image-ids "$new_created_ami" | grep "Description" | awk '{print $2}' | sed -e 's/^"//' -e 's/,$//' -e 's/"$//')
+
+echo -e "Amazon Web Services - Copying new created AMI into account's regions:\n\n"
+echo -e "Current source region is: $aws_source_region.\n"
+echo -e "Latest created AMI ID is: $new_created_ami.\n"
+echo -e "Latest created AMI Name is: $ami_name.\n"
+
+# Now we start the process of copy the Kubenow AMI across all the other regions
+for reg in ${aws_regions[*]}; do
+
+  if [ -n "$new_created_ami" ] && [ "$reg" != "$aws_source_region" ]; then
```

Comment:
Hope the majority of the script is clearly commented. Only point of explanation might be at this line (n° 26), regarding the double if condition. I've decided to design it this way in case we would want/need to change the default AWS region in future (currently set an environmental variable in Travis CI). This way, if the `source_region` is the same as the current `reg` in the `for` loop, then it's skipped as we don't want to create conflict with the already created one by packer.